microsoft/testsuites/network/netinterface.py

```diff
@@ -50,15 +54,33 @@ class NetInterface(TestSuite):
         priority=1,
         requirement=simple_requirement(network_interface=Synthetic()),
     )
-    def validate_netvsc_reload(self, node: Node) -> None:
+    def validate_netvsc_reload(self, node: Node, log_path: Path) -> None:
         self._validate_netvsc_built_in(node)
         network_interface_feature = node.features[NetworkInterface]
         # Test loading and unloading netvsc driver
         test_count = 0
         while test_count < self.NETVSC_RELOAD_TEST_COUNT:
             test_count += 1
             # Unload and load hv_netvsc
-            network_interface_feature.reload_module()
+            try:
+                network_interface_feature.reload_module()
+            except Exception as identifier:
+                # Sometimes the ssh session is inactive, call node.close() and retry
+                # can solve this problem. In this case, if there is no panic, then
+                # raise PassedException
+                node.log.debug(identifier)
```

Comment:
add more information in the debug message to explain the error is ignorable, and it's trying again.

Reply:
Done